jaeger-core/src/test/java/com/uber/jaeger/TracerTagsTest.java

```diff
@@ -133,5 +117,9 @@ public void testTracerTagsZipkin() throws Exception {
         assertEquals("Expecting " + key + " for " + spanType, expectedValue, actualValue);
       }
     }
+    Map<String, ?> processTags = tracer.tags();
+    assertEquals(true, processTags.containsKey("jaeger.version"));
+    assertEquals(true, processTags.containsKey("hostname"));
+    assertEquals(true, processTags.containsKey("tracer.tag.str"));
```

Comment:
This test still doesn't make sense to me. The reason it was parameterized was to ensure that tracer tags are converted into span tags when the span is first-in-process (2 out of 3 parameterized test cases). Since that logic has moved to Zipkin converter, this test should be moved there and operate only on the converted Zipkin span. The test that exists there is not as comprehensive as this one.

We may add a separate test here that tracer-level tags do not make it to Span tags, as well as that `withTag` method of the TracerBuilder works (which probably exists already elsewhere).